tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist =
+    py27,py34,py35,py36,py37,pypy,pypy3,coverage
+
+[testenv]
+commands =
+    zope-testrunner --test-path=. []
+deps =
+    .[test]
+
+[testenv:pypy3]
+setenv = PYTHONWARNINGS = ignore::DeprecationWarning:distutils
```

Comment:
(I am mildly curious about this: first it was simplejson, now distutils?  What is this warning that's only shown on pypy3?)

Reply:
It looks something like this and spoils the doctest output on many occasions. But it does not seem to be present on travis. 

```
---------------------------------------------------------------------
File "/.../z3c.recipe.mkdir/.tox/pypy3/site-packages/z3c/recipe/mkdir/regression.txt", line 27, in regression.txt
Failed example:
    printx(system(join('bin', 'buildout')))
Expected:
    Installing mydir.
    mydir: created path: /sample-buildout/mypath1
    mydir:   mode 0750
    mydir: created path: /sample-buildout/mypath2
    mydir:   mode 0750
Got:
    Installing mydir.
    mydir: created path: /sample-buildout/mypath1
    mydir:   mode 0750
    mydir: created path: /sample-buildout/mypath2
    mydir:   mode 0750
    /home/USER/zope/z3c.recipe.mkdir/.tox/pypy3/lib-python/3/distutils/__init__.py:4: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
      import imp
```